samples/MvcSubAreaSample.Web/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@{
+    ViewData["Title"] = "Home Page";
+    Layout = "/Views/Shared/_Layout.cshtml";
+}
+
+<div class="row">
+    <div class="col-md-3">
+        <h2>Areas</h2>
+        <ul>
+            <li><a href="/Restaurant">Restaurant</a></li>
```

Comment:
I ended up doing the taghelper style of these instead since it looks a bit cleaner to me and I was already using them elsewhere, so mixing the two seemed weird. If there's a reason this specifically should be @Html.Actionlink let me know.